platform/source/mbed_retarget.cpp

```diff
@@ -208,8 +215,83 @@ short DirectSerial::poll(short events) const
     }
     return revents;
 }
+#else
+/** Class providing minimal UART communication functionalities
+ */
+class MinimalConsole {
+public:
+    MinimalConsole(PinName tx, PinName rx, int baud);
```

Comment:
You don't nneed to take tx & rx since for the console they are set to STDIO_UART_TX & STDIO_UART_RX.

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/51afce6721b21feff4019c27b53742b4e2d79872..6b68737783bd5d135e93e5efb0479c0f91a7c0dc).